mods/ra/maps/training-camp/map.yaml

```diff
@@ -1058,9 +1058,10 @@ VoxelSequences:
 
 Weapons:
 	CrateNuke:
-		Warhead@areanuke2: PerCellDamage
+		Warhead@areanuke2: SpreadDamage
 			Damage: 250
-			Size: 4
+			Spread: 1c0
+			Falloff: 100, 100, 100, 100, 100, 0
```

Comment:
You can capture the war factory in the middle and build a demo truck. I don't think we should put that much effort into fine-tuning the nuke warhead for this map. I doubt anyone ever plays this map. Like, at all.